.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  Assemble:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Build project
+        run: ./gradlew clean build
+      - name: Build Docker image
+        run: docker build . -t paymenthubee.mifos.io/phee/importer-es
```

Comment:
Same goes here, is this pointing to a CR?

Reply:
Nah, just placeholder. Feel free to suggest an alternative